mrmustard/lab_dev/circuit_components.py

```diff
@@ -701,6 +697,7 @@ class CCView(CircuitComponent):
 
     @property
     def short_name(self):
+        """A shortened version of the component name."""
```

Comment:
```suggestion
        r"""
        A shortened version of the component name.
        """
```

Reply:
is this preferred for any particular reason?

Reply:
Mostly just personal preference I find starting on a new line makes things look marginally cleaner. But I'm happy as long as all the docs are consistent between each other 😄